metricbeat/module/consul/agent/data.go

```diff
@@ -0,0 +1,140 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package agent
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/elastic/beats/libbeat/common"
+)
+
+type valueHelper struct {
+	Value string
+	Unit  string
+}
+
+var (
+	allowedValues = map[string]valueHelper{
+		"consul.autopilot.healthy":         {Value: "autopilot.healthy"},
+		"consul.runtime.alloc_bytes":       {Value: "runtime.alloc", Unit: ".bytes"},
+		"consul.runtime.total_gc_pause_ns": {Value: "runtime.garbage_collector.pause.total", Unit: ".ns"},
+		"consul.runtime.gc_pause_ns":       {Value: "runtime.garbage_collector.pause.current", Unit: ".ns"},
+		"consul.runtime.total_gc_runs":     {Value: "runtime.garbage_collector.runs"},
+		"consul.runtime.num_goroutines":    {Value: "runtime.goroutines"},
+		"consul.runtime.heap_objects":      {Value: "runtime.heap_objects"},
+		"consul.runtime.sys_bytes":         {Value: "runtime.sys", Unit: ".bytes"},
+		"consul.runtime.malloc_count":      {Value: "runtime.malloc_count"},
+	}
+	allowedDetailedValues = map[string]valueHelper{
+		//"consul.raft.apply":      {Value: "raft.apply"},
+		//"consul.raft.commitTime": {Value: "raft.commit_time", Unit: ".ms"},
+	}
```

Comment:
Remove unused code?

Reply:
I prefer to leave it, as they are the values that we have removed. Will you be ok if I add a comment like "fields to be added, removed temporarly".